tools/system_test_jobs.groovy

```diff
@@ -0,0 +1,80 @@
+// Driver list
+DRIVERS = [ "nidcpower", "nidigital", "nidmm", "nifgen", "niscope", "niswitch", "nise", "nimodinst" ]
```

Comment:
no nitclk?

Reply:
Nope. nitclk needs another driver, so when #1195 is fixed, it will most likely involve adding tests to one of the other driver system_tests